### PR TITLE
Add conventions for types, study visits and death

### PR DESCRIPTION
**Pull request by MaximMoinat**
@Anne0507 @Spayralbe Please take a look at let me know if the text is clear enough.

### CODE REVIEW

**Review by Anne0507 — APPROVED** [review]

**Comment by Anne0507** [comment]
Looks good

File: docs/conventions.md [hunk 1]
```diff
@@ -54,3 +54,35 @@ Due to privacy reasons, the exact care site and location values are not mapped t
 ### CDM Source
 The name and location of the dataset are not captured in the `cdm_source` table.
 A record is created in this table, with the ETL date, source extraction date, ETL reference, vocab version and OMOP CDM version.
+
+### Death date
```
Comment (Spayralbe) [inline docs/conventions.md:58]: Just wondering: do we want to mention mapping to the observation table as well?

File: docs/conventions.md [hunk 2]
```diff
@@ -54,3 +54,35 @@ Due to privacy reasons, the exact care site and location values are not mapped t
...
+1. Create a record in the death table with at least the `person_id` and `death_date`. 
+   This may also include the primary cause of death.
+2. Update the person record and set the `death_datetime`. 
+   If no time granularity available, use _00:00:00_ as the time component.
+
+Contributory causes of death can be stored in the CONDITION table.
+
+### Type concept
+Every event table (e.g. condition_occurrence, observation) has a `_type_concept` field, which stores the record provenance.
+The main purpose to distinguish reliability of the record. A few types with increasing reliability:
```
Comment (Spayralbe) [inline docs/conventions.md:72]:
```suggestion
The main purpose is to determine the reliability of the record. A few types with increasing reliability:
```

**Review by Spayralbe — COMMENTED** [review]
✨

File: docs/conventions.md (hunk 1 above)
Comment (MaximMoinat) [reply docs/conventions.md:58]: No. Upon rereading the OMOP conventions on the death table, I do not think this is required.
> This field is the death date to be used in analysis, as determined by the ETL logic. Any additional information about a Person’s death is stored in the OBSERVATION table with the concept_id 4306655 or in the CONDITION_OCCURRENCE .
https://ohdsi.github.io/CommonDataModel/cdm60.html#PERSON

Comment (MaximMoinat) [reply docs/conventions.md:58]: But of course it will not hurt to leave it in for AB.

File: docs/conventions.md (hunk 2 above)
Comment (MaximMoinat) [reply docs/conventions.md]: Awesome, thanks!

**Pull request merged**